lib/action_controller/serialization.rb

```diff
@@ -56,7 +56,7 @@ def use_adapter?
 
     [:_render_option_json, :_render_with_renderer_json].each do |renderer_method|
       define_method renderer_method do |resource, options|
-        options.fetch(:serialization_context) { options[:serialization_context] = ActiveModelSerializers::SerializationContext.new(request) }
+        options[:serialization_context] = ActiveModelSerializers::SerializationContext.new(controller: self) unless options.key?(:serialization_context)
```

Comment:
:-1: I'd rather do it like 0.9 and not drag an enormous object around

see https://github.com/rails-api/active_model_serializers/blob/0-9-stable/lib/action_controller/serialization.rb#L61-L93